src/lib/spacefinder/liveblog-right-adverts.ts

```diff
@@ -0,0 +1,75 @@
+import type { Advert } from 'lib/dfp/Advert';
+import { createAdvert } from 'lib/dfp/create-advert';
+import { dfpEnv } from 'lib/dfp/dfp-env';
+import { displayAds } from 'lib/dfp/display-ads';
+import { displayLazyAds } from 'lib/dfp/display-lazy-ads';
+import { queueAdvert } from 'lib/dfp/queue-advert';
+import { isInEagerPrebidVariant } from 'lib/experiments/eager-prebid-check';
+import { requestBidsForAds } from '../header-bidding/request-bids';
+
+type AdsInsertedCustomEventDetail = {
+	numAdsToInsert: number;
+	fromIndex: number;
+};
+
+type AdsInsertedCustomEvent = {
+	detail: AdsInsertedCustomEventDetail;
+};
+
+const isCustomEvent = (event: Event): event is CustomEvent => {
```

Comment:
A function of type (e: CustomEvent) => void is not a valid instance of EventListener, so I had to use Event then use this type guard

https://stackoverflow.com/questions/47166369/argument-of-type-e-customevent-void-is-not-assignable-to-parameter-of-ty/47171216#47171216